README.md

```diff
@@ -234,38 +215,6 @@ Here are the addresses on those networks:
 
 </details>
 
-<details>
```

Comment:
Did you mean to remove configs for BSC as well?

Reply:
Actually yes, I don't know why but it's a duplicate of the lines just above 🤷‍♂️